### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Prasiddha777**
Landing Page Ui Review

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,34 +1,21 @@
...
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </div>
-  )
+    <BrowserRouter>
+      <MasterLayout>
```
Comment (alsongarbuja) [inline src/App.tsx:10]: Yo master layout jun xa tyo chai sabai page mah use hudaina hai

use hune bhaneko landing page raw about page mah mattra ho

docs, install, login haru sabai mah arkai layout hunxa pheri

File: tailwind.config.cjs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        primary: "#535BF2 ",
+        secondary: "#FFAB12",
+        neutralDark: "#131515",
+        danger: "#DA4944",
+        success: "#4F772D",
+        textfield: "B9BCFF",
+      },
+      fontFamily: {
+        sen: ["Sen", "sans-serif"],
+        lato: ["Lato", "sans-serif"],
```
Comment (alsongarbuja) [inline tailwind.config.cjs:17]: yo hamro lato hoina arko font use garne bhane thim ni tyo use hana hai

File: src/components/Navbar.tsx
Comment (alsongarbuja) [inline src/components/Navbar.tsx:1]: Yo chai completely change garne parne raixa responsive navbar

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      {/* footer end section */}
+
+      <div className="flex pt-9 pb-0 sm:pt-14 sm:justify-between items-center font-lato">
+        <div className="text-white text-sm tracking-widest">
+          <p>©2023 POLARIS SUITE. ALL RIGHTS RESERVED</p>
+        </div>
+        <div className="flex space-x-2 sm:space-x-16">
+          <img src={github_logo} alt="github" className="w-7 h-7 " />
+          <img src={twitter_logo} alt="twitter" className="w-7 h-7 " />
+          <img src={insta_logo} alt="insta" className="w-7 h-7" />
```
Comment (alsongarbuja) [inline src/components/Footer.tsx:103]: Yo insta ko logo purai na ako ho ki yestai ho

File: src/assets/images/insta_logo.png
Comment (alsongarbuja) [inline src/assets/images/insta_logo.png:1]: Yo insta icon mileko ho??

**Review by alsongarbuja — CHANGES_REQUESTED** [review]

File: src/components/Navbar.tsx
Comment (Prasiddha777) [reply src/components/Navbar.tsx:1]: Changed the layout of the responsive navbar for small screens

File: src/assets/images/insta_logo.png
Comment (Prasiddha777) [reply src/assets/images/insta_logo.png:1]: New Icon downloaded and changed

File: src/components/Footer.tsx (hunk 3 above)
Comment (Prasiddha777) [reply src/components/Footer.tsx:105]: tyo bahira ko sqaure matra download vayeko rahecha suru maw so now resolved

**Review by alsongarbuja — APPROVED** [review]

**Review by Salipa-Gurung — APPROVED** [review]

**Pull request merged**